packages/teleterm/src/ui/TopBar/Clusters/ConfirmClusterChangeDialog.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Text, ButtonIcon, ButtonWarning } from 'design';
+import DialogConfirmation, {
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+} from 'design/DialogConfirmation';
+import { Close } from 'design/Icon';
+
+const changeSelectedClusterWarning =
+  'Resources from different clusters cannot be combined in an access request. Current items selected will be cleared. Are you sure you want to continue?';
+
+export default function ChangeResourceDialog({
```

Comment:
This component is used with a name `<ConfirmChangeDialog>` and sits inside  `ConfirmClusterChangeDialog.tsx`, that's a bit confusing :p 
Using one name will improve readability.